platform/commonUI/general/res/templates/controls/time-controller.html

```diff
@@ -19,84 +19,90 @@
  this source code distribution or the Licensing information page available
  at runtime from the About dialog for additional information.
 -->
-<!-- MINE -->
 <div ng-controller="TimeRangeController">
-	<div class="l-time-range-inputs-holder">
-		<span class="l-time-range-inputs-elem ui-symbol type-icon">&#x43;</span>
-	    <span class="l-time-range-input" ng-controller="ToggleController as t1">
-	        <!--<span class="lbl">Start</span>-->
-		    <span class="s-btn time-range-start" ng-click="t1.toggle()">
-		        <span class="val">{{startOuterText}}</span>
-	            <a class="ui-symbol icon icon-calendar"></a>
-	            <mct-popup ng-if="t1.isActive()">
-		            <div mct-click-elsewhere="t1.setState(false)">
-			            <mct-control key="'datetime-picker'"
-			                         ng-model="ngModel.outer"
-			                         field="'start'"
-			                         options="{ hours: true }">
-			            </mct-control>
-		            </div>
-	            </mct-popup>
-		    </span>
-	    </span>
+    <div class="l-time-range-inputs-holder">
+        <span class="l-time-range-inputs-elem ui-symbol type-icon">&#x43;</span>
+        <span class="l-time-range-input" ng-controller="ToggleController as t1">
+            <!--<span class="lbl">Start</span>-->
+            <span class="s-btn time-range-start">
+                <input type="text"
+                       ng-model="boundsModel.start"
+                       ng-class="{ error: !boundsModel.startValid }">
+                </input>
+                <a class="ui-symbol icon icon-calendar" ng-click="t1.toggle()"></a>
+                <mct-popup ng-if="t1.isActive()">
+                    <div mct-click-elsewhere="t1.setState(false)">
+                        <mct-control key="'datetime-picker'"
```

Comment:
It appears I misunderstood the initial datepicker implementation.  I was under the impression that this entire component:
![picker](https://cloud.githubusercontent.com/assets/226503/10705175/fa5ed8f0-7990-11e5-899c-e32fe92315e1.png)
was a single mct-control, such that it could be used within a view (in this case, time conductor) to abstract out all of the work required to display a date, support typing, and handle validation/etc.

This would drastically simplify the markup in this template and make it much easier to reuse this component.  It would also move the complexity of managing user inputs from the timerangecontroller to the individual mct-control, which would be highly desirable.